flare/framework/manager.py

```diff
@@ -20,24 +20,29 @@
 
 class Manager(object):
     def __init__(self, ct_settings):
+        """
+            Initialize `Manager`. `ct_settings` is used to create
+            `ComputationTask`; The parameters in `ct_settings` are for each 
+            `ComputationTask`.
+        """
         self.agents = []
         self.cts = {}
         self.state_specs = {}
-        self.wrappers = {}
+        self.CDPs = {}
         for name, setting in ct_settings.iteritems():
             self.cts[name] = ComputationTask(name, **setting)
             self.state_specs[name] = self.cts[name].get_state_specs()
-            self.wrappers[name] = self.cts[name].wrapper
+            self.CDPs[name] = self.cts[name].CDP
         self.logger = GameLogger(1, 100)
 
     def add_agent(self, agent):
         agent.id = len(self.agents)
         # `Agent` needs to know the state specs to prepare state data
         agent.make_initial_states(self.state_specs)
         self.agents.append(agent)
-        for name, wrapper in self.wrappers.iteritems():
-            agent.add_helper(
-                wrapper.create_helper(agent.id), agent.pack_exps,
+        for name, cdp in self.CDPs.iteritems():
+            agent.add_agent_helper(
```

Comment:
My original comment was meant to change cdp.create_helper to cdp.create_agent_helper, not agent.add_helper